Changed logger verbosity levels

keegansmith21 opened the pull request:
I got sick of seeing so many warnings all the time so i shut them all up

NickSwainston reviewed (commented):
This is all fine but I think it would be best to have a single warning that says something like "This script uses several approximations if there isn't sufficient data, use -L DEBUG if you would like these approximations to be output" just so it's clear to the user how to display more imformation if they're unsure about an uncertainty

Since you're no longer printing out any progress by default you could implement a progress bar to display what percentage of the pulsars you've finished calculating. This is optional though, just thought it might be cool

keegansmith21 commented:
> This is all fine but I think it would be best to have a single warning that says something like "This script uses several approximations if there isn't sufficient data, use -L DEBUG if you would like these approximations to be output" just so it's clear to the user how to display more imformation if they're unsure about an uncertainty
> 
> Since you're no longer printing out any progress by default you could implement a progress bar to display what percentage of the pulsars you've finished calculating. This is optional though, just thought it might be cool

I've added a message when using the multi-processing option. Also added a progress bar

keegansmith21 commented:
Moved the progress bar module to the correct "vcstools" module directory
